CondCore/SiPhase2TrackerPlugins/BuildFile.xml

```diff
@@ -0,0 +1,9 @@
+<use name="CondCore/ESSources"/>
+<use name="CondFormats/CondTest"/>
+<use name="CondFormats/DataRecord"/>
+<use name="CondFormats/Serialization"/>
+<use name="CondFormats/SiPhase2TrackerObjects"/>
+<use name="rootrflx"/>
+<use name="boost"/>
+<use name="tbb"/>
```

Comment:
for my own education what drives this dependency:
` <use name="tbb"/>`?

Reply:
That is the Thread Building Blocks library. Some time passed by now, but I remember ld was throwing a hissy fit without it. It was a linker error so deep in terms of dependency that I just threw the dependency in the Buildfile and accepted it as god-given. Let me test again.

Reply:
Tested, and tbb can be safely removed